components/centraldashboard/app/server.ts

```diff
@@ -1,51 +1,15 @@
-import express from "express";
-import { resolve } from "path";
-import process from "process";
+import express from 'express';
+import process from 'process';
```

Comment:
Wait why?

Reply:
We moved the configuration of the static routes to the routes module so resolve is no longer needed here.

Reply:
Oh sorry, I meant why are we importing `process`? It's a node.js global

Reply:
Good point. I think I was just expecting the Typescript compiler to complain if I didn't include it and then I referenced a global library. I removed it.